go/storage/lock/lock_manager.go

```diff
@@ -9,10 +13,18 @@ type LockManager interface {
 	Release(version int64, success bool) error
 }
 
-type EmptyLockManager struct{}
+type EmptyLockManager struct {
+	lock sync.Mutex
```

Comment:
Should not add a lock here. If your target is to test concrrent write, maybe it's better to write a memory lock manager.